docs/overview/code_of_conduct.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Code of Conduct
-weight: 8
+weight: 9
```

Comment:
Adjusted the ordering of the pages as I added a page on performance.

But TBH I'm thinking of using a different way to set the order of the pages, because currently it means that if I want to a page to the middle, I have to update  the weights for half of all the pages.

Ideally the order could be set on the file name, e.g. `1--code_of_conduct.md`. But file names are part of the final URL, so the URLs wouldn't look pretty.

So instead I'm thinking to set the ordering in `SUMMARY.md`, which defines the site navigation.

Reply:
Is it possible to use deciment points? A simple way would then be to just use 7.5 to place a page between 7 and 8.

Reply:
That's a good point, but it still requires you to know which files have weight 7 and weight 8, so you still need to look through all the files to find them 😄